oslat-client

```diff
@@ -130,7 +150,7 @@ if [ $? != 0 ]; then
     exit_error "${output}"
 fi
 
-cmd="taskset -c ${HK_CPUS},${WORKLOAD_CPUS} oslat --duration ${duration} ${rtprio_opt} --cpu-list ${WORKLOAD_CPUS} --cpu-main-thread ${cpu_main_thread} ${trace_args}"
+cmd="taskset -c ${WORKLOAD_CPUS} oslat --duration ${duration} ${rtprio_opt} --cpu-list ${LATENCY_CPUS} --cpu-main-thread ${cpu_main_thread} ${trace_args}"
```

Comment:
Does `--cpu-main-thread` accept more than one CPU?  That could happen in the default case...

Reply:
we can fix this by adding a new param "cpu-main-thread". Will keep the default as the hk cpus then.